<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDokter</class>
 <widget class="QWidget" name="LoginDokter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Login Dokter - Klinik Sehat</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{background:#F5F7FA}QLineEdit{background:white;border:2px solid #E8E8E8;border-radius:8px;padding:12px;color:#2C3E50}QLineEdit:focus{border-color:#2E86DE}QPushButton{background:#2E86DE;color:white;border:none;border-radius:8px;padding:14px;font-weight:bold}QPushButton:hover{background:#54A0FF}QPushButton:pressed{background:#1E6FBE}QLabel#lblRegister{color:#7F8C8D}QLabel#lblRegisterLink{color:#2E86DE;text-decoration:underline}</string>
  </property>
  <layout class="QHBoxLayout" name="mainLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="leftPanel" native="true">
     <property name="styleSheet">
      <string notr="true">background:qlineargradient(x1:0,y1:0,x2:0,y2:1,stop:0 #2E86DE,stop:1 #54A0FF);</string>
     </property>
     <layout class="QVBoxLayout" name="leftLayout">
      <property name="spacing">
       <number>24</number>
      </property>
      <property name="leftMargin">
       <number>60</number>
      </property>
      <property name="topMargin">
       <number>60</number>
      </property>
      <property name="rightMargin">
       <number>60</number>
      </property>
      <property name="bottomMargin">
       <number>60</number>
      </property>
      <item>
       <spacer name="topSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="lblIconBig">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>120</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>120</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>72</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background:rgba(255,255,255,0.2);border-radius:60px;</string>
        </property>
        <property name="text">
         <string>üè•</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblKlinik">
        <property name="font">
         <font>
          <pointsize>32</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color:white;</string>
        </property>
        <property name="text">
         <string>Klinik Sehat</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblTagline">
        <property name="font">
         <font>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color:rgba(255,255,255,0.9);</string>
        </property>
        <property name="text">
         <string>Sistem Informasi Manajemen Klinik</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="divider">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>2</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>2</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background:rgba(255,255,255,0.3);</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblInfo">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color:rgba(255,255,255,0.85);</string>
        </property>
        <property name="text">
         <string>Melayani dengan sepenuh hati
untuk kesehatan Anda</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="bottomSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="rightPanel" native="true">
     <property name="styleSheet">
      <string notr="true">background:white;</string>
     </property>
     <layout class="QVBoxLayout" name="rightLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <spacer name="topSpacerRight">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QWidget" name="loginContainer" native="true">
        <property name="minimumSize">
         <size>
          <width>400</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>480</width>
          <height>16777215</height>
         </size>
        </property>
        <layout class="QVBoxLayout" name="loginLayout">
         <property name="spacing">
          <number>28</number>
         </property>
         <property name="leftMargin">
          <number>60</number>
         </property>
         <property name="topMargin">
          <number>40</number>
         </property>
         <property name="rightMargin">
          <number>60</number>
         </property>
         <property name="bottomMargin">
          <number>40</number>
         </property>
         <item>
          <layout class="QVBoxLayout" name="headerLayout">
           <property name="spacing">
            <number>12</number>
           </property>
           <item>
            <widget class="QLabel" name="lblWelcome">
             <property name="font">
              <font>
               <pointsize>28</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color:#2E86DE;</string>
             </property>
             <property name="text">
              <string>Selamat Datang</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lblSubtitle">
             <property name="font">
              <font>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color:#7F8C8D;</string>
             </property>
             <property name="text">
              <string>Login sebagai dokter untuk mengakses sistem</string>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QFrame" name="separator">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>1</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>1</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background:#E8E8E8;</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="formLayout">
           <property name="spacing">
            <number>20</number>
           </property>
           <item>
            <layout class="QVBoxLayout" name="usernameLayout">
             <property name="spacing">
              <number>10</number>
             </property>
             <item>
              <widget class="QLabel" name="lblUsername">
               <property name="font">
                <font>
                 <pointsize>11</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">color:#2C3E50;</string>
               </property>
               <property name="text">
                <string>Username</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="txtUsername">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>52</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>11</pointsize>
                </font>
               </property>
               <property name="placeholderText">
                <string>Masukkan username Anda</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="passwordLayout">
             <property name="spacing">
              <number>10</number>
             </property>
             <item>
              <widget class="QLabel" name="lblPassword">
               <property name="font">
                <font>
                 <pointsize>11</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">color:#2C3E50;</string>
               </property>
               <property name="text">
                <string>Password</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="txtPassword">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>52</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>11</pointsize>
                </font>
               </property>
               <property name="echoMode">
                <enum>QLineEdit::Password</enum>
               </property>
               <property name="placeholderText">
                <string>Masukkan password Anda</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="formSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>10</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="btnLogin">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>56</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>13</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(46, 134, 222);</string>
           </property>
           <property name="text">
            <string>Login</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="registerLayout">
           <property name="spacing">
            <number>6</number>
           </property>
           <item>
            <spacer name="leftRegisterSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="lblRegister">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Belum punya akun?</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lblRegisterLink">
             <property name="font">
              <font>
               <underline>true</underline>
              </font>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="text">
              <string>Daftar di sini</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="rightRegisterSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="bottomSpacerRight">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
